scripts/helpers/simple.prompt.ts

```diff
@@ -0,0 +1,111 @@
+import * as inquirer from 'inquirer';
```

Comment:
What is this used for?

Reply:
For being a wrapper around the "inquirer" library to make it super-simple to have commandline prompts.
I borrowed it from a different project (publish tools) where it's used.  So some of the methods aren't relevant to us for now.  But they might be at some point (e.g., if we automated the deployment and have a commandline, we might want to give you a choice list for something).

Reply:
Still, sounds like something that isn't needed.. at least yet